src/main/java/org/ow2/chameleon/core/Constants.java

```diff
@@ -89,6 +89,12 @@
      */
     public static final String CHAMELEON_AUTO_REFRESH = "chameleon.auto.refresh";
 
+    /**
+     * The property used to allowed chameleon keep running if it starts in interactive mode and shelbie disapear.
+     * By defaut , false
+     */
```

Comment:
Change to: 
This property is used to keep chameleon running if the interactive mode is enabled and shelbie is stopped. {@code false} by default.